lib/tasks/db_load.rake

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+namespace :db do
```

Comment:
let's namespace this to `tags`?

Reply:
I used this namespace as in the context of a rails script run from the console it is to do with the DB. It creates the `db:load:...` namespace that loads data into the database. It is used as a place to put one off scripts for... loading data into the database. I think it is clearer as it is. Are we going to have a `partners` namespace, a `partnerships`, a `calendars` ? this will clutter up the rails task output and make it harder to find (or guess at) that useful data loading script

Reply:
But this isn't setting up a generic database which is what the `db` namespace is for - its our specific seed data for our specific instance of PlaceCal. It clashes with the existing Rails namespace. Other people using it would prob not use our categories, or at least we should decouple it

How about we start a namespace just for this? `pht` or something?

I do think they need to be seeded individually yes - we need granular control as once added we won't be deleting them. One big `load` task seems bad to me.

Reply:
i would suggest then moving all of our scripts into the `placecal` namespace.

- move only this script (adhoc, need to move rest later, making more issue chaff)
- move all scripts (feature creep, would have to update all scripts, would have to update docs)